### DISCUSSION TITLE
Is there a Capacitor equivalent for cordova-plugin-background-geolocation ?

### DISCUSSION THREAD

**Discussion by chris4long** [ionic-team/capacitor#3626] [Q&A]
I'm writing an app (React with Capacitor) for iOS and Andriod, with a geolocation feature which needs to run in the background, The above plugin (cordova-plugin-background-geolocation) allowed background updates, but I can't see any equivalent in Capacitor?

**Comment by imhoffd** [comment]
You can look at the Background Geolocation community plugin proposal: https://github.com/capacitor-community/proposals/issues/40
